sympy/integrals/manualintegrate.py

```diff
@@ -342,7 +342,10 @@ def _parts_rule(integrand, symbol):
     # log, inverse trig, algebraic (polynomial), trigonometric, exponential
     def pull_out_polys(integrand):
         integrand = integrand.together()
-        polys = [arg for arg in integrand.args if arg.is_polynomial(symbol)]
+        polys = [arg for arg in integrand.args if (arg.is_polynomial(symbol) or \
+                                                   (isinstance(arg, sympy.Pow) and \
```

Comment:
It may be useful to include expressions of algebraic type, but I'd expect them to be of the form `polynomial**rational`. If the base contains transcendental functions or the exponent is irrational, the power cannot be considered algebraic. Powers with exponent -1 are also algebraic (even rational). It is not clear why they should not be allowed.